tests/resources/rebuild_example_data

```diff
@@ -29,28 +59,57 @@ Remember to run this with the other supported versions of pandas!
 """
 
 
+def changed_results(new, old):
+    """
+    Get the names of results that changed since the last invocation.
+
+    Useful for verifying that only expected results changed.
+    """
+    changed = []
+    for col in new:
+        if col not in old:
+            changed.append(col)
+            continue
+        try:
+            pd.util.testing.assert_frame_equal(
+                new[col][examples._cols_to_check],
+                old[col][examples._cols_to_check],
```

Comment:
it would be nice to report it as different instead of crashing. There may be differences in other examples _and_ a test was removed.